services/github-service-api/src/main/java/org/kontinuity/catapult/service/github/api/GitHubRepository.java

```diff
@@ -12,7 +13,7 @@
     /**
      * @return the "git" protocol transport URL
      */
-    String getGitTransportUrl();
+    URI getGitTransportUrl();
```

Comment:
that being said, I approve the idea of removing that particular method if it was only used for logging purposes ;-)